packages/app-layout/src/vaadin-app-layout.js

```diff
@@ -472,6 +466,8 @@ class AppLayout extends ElementMixin(ThemableMixin(ControllerMixin(PolymerElemen
     });
     this.$.drawer.addEventListener('transitionend', () => {
       this.__isDrawerAnimating = false;
+      this.__updateOffsetSizePending = false;
+      this._updateOffsetSize();
```

Comment:
Shouldn't we call `_updateOffsetSize` only if `__updateOffsetSizePending` is true? Otherwise, `__updateOffsetSizePending` doesn't seem to be useful.

Reply:
Done.